### ISSUE TITLE
consider upgrading to the latest version of dbt_utils

### ISSUE THREAD

**Issue by markns** [Montreal-Analytics/dbt-snowflake-utils#33]
Hi, thanks for the helpful repo! 🙌 

Would you consider upgrading to the latest version of dbt_utils - https://hub.getdbt.com/dbt-labs/dbt_utils/1.1.0/? 
```
packages:
  - package: dbt-labs/dbt_utils
    version: [">=0.7.0", "<1.2.0"]
```

**Comment by jasonmaddernstudylink** [comment]
This would be great - it's tripping up other dependencies across my packages at present and I needed to downgrade others

**Comment by jakobfloe** [comment]
+1 for this one.

**Comment by smithbc25** [comment]
+1

**Comment by alfredodimassimo** [comment]
Hi everyone, this issue has been resolved in the recent [PR#40](https://github.com/Montreal-Analytics/dbt-snowflake-utils/pull/40#issue-1799634986). Please let us know if any other issues arise with regards to this ticket.
